### DISCUSSION TITLE
Practica CSS feedback

### DISCUSSION THREAD

**Discussion by AngelCruzO** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#307] [Show and tell]
Hola a todos, no suelo pedir feedback sobre mis trabajos, pero en esta ocasión lo agradeceria demasiado, ya que lo considero un logro para mi al realizarlo con puro CSS.

Aclaro que la pagina no esta optimizada para el tema responsive, solo para desktop.

[Practica CSS](https://angelcruzo.github.io/vaccination-COVID/)

De antemano muchas gracias y que tengan un gran dia

**Comment by JuanSalvador73** [comment]
En general va muy bien. Si, veo que la parte de "responsive" sigue en desarrollo, literalmente ya acabaste el trabajo. El uso de div uno sobre otro usaste Z-index, seguramente.

**Reply by AngelCruzO** [reply]
Si para la primera sección utilice z-index, para mandar la imagen al fondo y se viera el form encima. Ya para la parte de responsiva, solo es agregar los media querys
